homeassistant/components/ecobee/switch.py

```diff
@@ -88,3 +96,41 @@ async def async_turn_off(self, **kwargs: Any) -> None:
             self.data.ecobee.set_ventilator_timer, self.thermostat_index, False
         )
         self.update_without_throttle = True
+
+
+class EcobeeSwitchAuxHeatOnly(EcobeeBaseEntity, SwitchEntity):
+    """Representation of a aux_heat_only ecobee switch."""
+
+    _attr_has_entity_name = True
+    _attr_name = "Aux Heat Only"
```

Comment:
Two comments on the name:

- The name should the added to the `strings.json` file instead and set a translation key on this entity. This will the entity name translatable.
- Entity names should be sentence style capitalized. So, in this case `Aux heat only`.

Reply:
Done. Thanks.